coldfront/core/billing/views/admin_views.py

```diff
@@ -308,3 +313,43 @@ def get_context_data(self, **kwargs):
         context['next_url_parameter'] = urlencode({'next': next_url})
 
         return context
+    
+class BillingIDValidateView(LoginRequiredMixin, UserPassesTestMixin, FormView):
+
+    form_class = BillingIDValidateManyForm
+    template_name = 'billing/billing_id_validate.html'
+
+    def test_func(self):
+        return self.request.user.is_superuser
+
+    def form_valid(self, form):
+        billing_ids = form.cleaned_data.get('billing_ids')
+
+        validation_results = ''
+        if billing_ids:
+            separated_billing_ids = [s.strip() for s in billing_ids.split('\n')]
+            exists_malformed_or_invalid = False
+            for full_id in separated_billing_ids:
+                if is_billing_id_well_formed(full_id):
+                    billing_activity = get_billing_activity_from_full_id(full_id)
```

Comment:
There are many billing IDs not stored in our database. It's sufficient to pass `full_id` to the validity function.